src/file.jl

```diff
@@ -194,8 +194,8 @@ function File(source;
     dateformat::Union{String, Dates.DateFormat, Nothing}=nothing,
     dateformats::Union{AbstractDict, Nothing}=nothing,
     decimal::Union{UInt8, Char}=UInt8('.'),
-    truestrings::Union{Vector{String}, Nothing}=nothing,
-    falsestrings::Union{Vector{String}, Nothing}=nothing,
+    truestrings::Union{Vector{String}, Nothing}=["true", "True", "TRUE"],
```

Comment:
What's the point of still allowing `nothing` if it's neither the default nor documented?

Reply:
I'm not sure if this is undefended, but you can still provide nothing and get default behaviour . But probably changing the type here is better. Should I do it?

Reply:
The effect of passing `nothing` is actually determined by Parsers.jl:
https://github.com/JuliaData/Parsers.jl/blob/545326062542da983a2121236f176b0f5ccc5714/src/Parsers.jl#L18-L19

I'd say we should either disallow passing `nothing` here, or change Parsers.jl to use the same default as here. AFAICT, the point of passing `nothing` is that it uses [an optimized path](https://github.com/JuliaData/Parsers.jl/blob/c7199940a0d412ae9b72e8f933fa5909fe56ddf2/src/bools.jl#L8) for performance: the default values for CSV.jl should definitely be optimized for, so maybe changing Parsers.jl is the best option @quinnj?